src/popup/components/TransferSendForm.vue

```diff
@@ -53,7 +53,7 @@
       v-validate="{
         required: true,
         not_same_as: isMultisig? multisigVaultAddress : account.address,
-        name_registered_address_or_url: true,
+        name_registered_address: true,
```

Comment:
Shouldn't we still be able to tip superhero.com related urls?

Reply:
From the UI: no. We are only leaving some logic to allow tipping of superhero URLs done on the superhero.com side.